integrations/google-calendar/actions/whoami.ts

```
@@ -0,0 +1,21 @@
+import type { UserInfoResponse } from '../types';
+
+export default async function runAction(nango: NangoAction): Promise<UserInformation> {
+    const config: ProxyConfiguration = {
+        // https://cloud.google.com/identity-platform/docs/reference/rest/v1/UserInfo
+        endpoint: '/oauth2/v1/userinfo',
+        params: {
+            alt: 'json'
+        }
+    };
+
+    const { data } = await nango.get<UserInfoResponse>(config);
```

ESLint

[error] Nango API calls should include a retries property in the options object

(@nangohq/custom-integrations-linting/proxy-call-retries)